admin/class-automation.php

```diff
@@ -43,6 +43,46 @@ class Automation {
 			'location' => 'article_metadata',
 			'type'     => 'boolean',
 		],
+		'Section'     => [
+			'location' => 'apple_news_api',
+			'type'     => 'string',
+		],
+		'Slug'        => [
```

Comment:
Set this key to `slug.#text#` and move `Slug` to a new `label` property with translation.
```suggestion
		'slug.#text#'        => [
```

Reply:
Only hitch here is that the parser/compiler won't let me use an expression (`__()` translation) inside a constant (`const FIELDS`).  For now I've left them untranslated but I'm guessing there's a way to convert this to a local class variable or something else in PHP that'd allow expressions.

Reply:
Good call - we can convert that to a normal property instead of a constant (so just `public array $fields = [ // etc`).